src/components/Table/Table.stories.tsx

```diff
@@ -123,7 +157,26 @@ const rows = [
   ),
 ];
 
+const MobileView = () => {
+  const mobile: boolean = useMediaQuery(`(max-width: ${tokens.BreakpointsSm})`);
+  if (mobile) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+const DecideLayout = () => {
+  const mobile: boolean = MobileView();
+  if (mobile) {
+    return ScreenSize.Mobile;
+  } else {
+    return ScreenSize.Laptop;
+  }
+};
```

Comment:
Commenting here to say the same thing I said in the `app-frontend-react` reviews. These two functions could be two lines of code.